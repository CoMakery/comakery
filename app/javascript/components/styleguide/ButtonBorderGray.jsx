import React from 'react'
import PropTypes from 'prop-types'
import classNames from 'classnames'
import Button from './Button'

class ButtonBorderGray extends React.Component {
  render() {
    const {className, ...other} = this.props

    const classnames = classNames(
      'button__border__gray',
      className
    )

    return (
      <React.Fragment>
        <Button className={classnames} {...other} />
      </React.Fragment>
    )
  }
}

ButtonBorderGray.propTypes = {
  className: PropTypes.string
}
ButtonBorderGray.defaultProps = {
  className: ''
}
export default ButtonBorderGray
