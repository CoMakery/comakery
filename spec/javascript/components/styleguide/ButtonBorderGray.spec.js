import React from 'react'
import { shallow } from 'enzyme'
import ButtonBorderGray from 'components/styleguide/ButtonBorderGray'

describe('ButtonBorderGray', () => {
  it('renders correctly without props', () => {
    const wrapper = shallow(<ButtonBorderGray />)

    expect(wrapper).toMatchSnapshot()
  })
})
