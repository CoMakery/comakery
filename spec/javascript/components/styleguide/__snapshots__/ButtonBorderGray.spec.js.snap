// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ButtonBorderGray renders correctly without props 1`] = `
<Fragment>
  <Button
    className="button__border__gray"
    disabled={false}
    type="button"
    value="publish"
  />
</Fragment>
`;
